onnx_coreml/converter.py

```diff
@@ -276,7 +276,7 @@ def _set_deprocessing(is_grayscale,  # type: bool
 
 
 def _prepare_onnx_graph(graph, transformers):  # type: (Graph, Iterable[Transformer]) -> Graph
-    graph = infer_shapes_and_types(graph)
+    # graph = infer_shapes_and_types(graph)
```

Comment:
If we are not using this function then I guess the file "_shapeinference.py" should be removed.
Can you do that as well in this PR?

Reply:
removed.

Reply:
Cool!